pyopae/pyshared_buffer.h

```diff
@@ -33,7 +33,8 @@
 const char *shared_buffer_doc();
 
 const char *shared_buffer_doc_allocate();
-
+opae::fpga::types::shared_buffer::ptr_t shared_buffer_allocate(
```

Comment:
The namespace for pointer type is so long. Is there a way to simplify it?

Reply:
You can shorten it (give it an alias) however it is not recommended to do in header files.